src/Diagnostics/Diagnostics.jl

```diff
@@ -4,8 +4,7 @@ import ClimaComms
 
 using ..Bucket: BucketModel
 
-using ClimaLand
-import ClimaLand: SoilCanopyModel  
+import ..SoilCanopyModel
```

Comment:
ah ha!

Reply:
The problem was that `Diagnostics.jl`  was imported before the soil module